common/static/js/vendor/pdfjs/viewer.js

```diff
@@ -4662,8 +4662,6 @@ var PDFViewerApplication = {
       presentationModeButton:
         document.getElementById('secondaryPresentationMode'),
       openFile: document.getElementById('secondaryOpenFile'),
-      print: document.getElementById('secondaryPrint'),
```

Comment:
I would prefer not to edit vendor files. As we discussed in standup, let's put this on hold for now until Shelby can catch up with Marco.